tests/test_dub_export_unique.py

```
@@ -100,6 +103,46 @@ async def _runner(*cmd, **_):
 
 class TestDubExportUniqueness:
+    @pytest.mark.skipif(
+        shutil.which("ffmpeg") is None or shutil.which("ffprobe") is None,
+        reason="ffmpeg and ffprobe are required for the container regression",
+    )
+    def test_default_export_marks_dubbed_audio_as_default(self, app_client):
+        """#1575: players that honor MP4 disposition must choose the dub."""
+        client, dc, _dx, tmp = app_client
+        job_id, job_dir = _seed_job_with_tracks(dc, tmp)
+        video_path = job_dir / "original.mp4"
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)